Dockerfile

```diff
@@ -85,16 +85,16 @@ RUN set -x \
 # plugin wasn't able to properly setup the net ns.
 # The bridge is based on the same commit as the one above.
 #ENV CNI_COMMIT 6bfe036c38c8e1410f1acaa4b2ee16f1851472e4
-ENV CNI_TEST_BRANCH custom-bridge
-RUN set -x \
-       && export GOPATH="$(mktemp -d)" \
-       && git clone https://github.com/runcom/plugins.git "$GOPATH/src/github.com/containernetworking/plugins" \
-       && cd "$GOPATH/src/github.com/containernetworking/plugins" \
-       && git checkout -q "$CNI_TEST_BRANCH" \
-       && ./build.sh \
-       && mkdir -p /opt/cni/bin \
-       && cp bin/bridge /opt/cni/bin/bridge-custom \
-       && rm -rf "$GOPATH"
+#ENV CNI_TEST_BRANCH custom-bridge
```

Comment:
Should we just remove these?  Why comment them out?

Reply:
Yup ... I just wanted to make sure I was right before deleting it ... Now deleted.